sdk/test/Services/S3Control/UnitTests/Custom/S3ControlArnTestUtils.cs

```diff
@@ -16,71 +16,28 @@
 using Amazon.Runtime.Internal;
 using Amazon.Runtime.Internal.Auth;
 using Amazon.Runtime.Internal.Transform;
-using Amazon.Runtime.Internal.Util;
 using Amazon.S3Control;
 using Amazon.S3Control.Internal;
-using System;
+using AWSSDK.UnitTests.Mocking;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using static AWSSDK.UnitTests.Mocking.TestUtils;
 
 namespace AWSSDK.UnitTests
 {
     public class S3ControlArnTestUtils
     {
         public static IRequest RunMockRequest(AmazonWebServiceRequest request, IMarshaller<IRequest, AmazonWebServiceRequest> marshaller, AmazonS3ControlConfig config)
         {
-            var pipeline = new RuntimePipeline(new List<IPipelineHandler>
+            var pipelineHandlers = new List<IPipelineHandler>
             {
                 new NoopPipelineHandler(),
+                new Signer(),
```

Comment:
Adding this so we can assert that the signing region in the `Authorization` header is correct in the new test.